src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/jira/StandaloneNodeFormula.kt

```diff
@@ -74,7 +74,8 @@ internal class StandaloneNodeFormula(
                     jiraIp = ssh.host.ipAddress
                 )
                 connection.execute("echo jira.home=`realpath $jiraHome` > $unpackedProduct/atlassian-jira/WEB-INF/classes/jira-application.properties")
-                connection.execute("echo jira.autoexport=false > $jiraHome/jira-config.properties")
+                connection.execute("sed -i '/^jira.autoexport=/d' $jiraHome/jira-config.properties")
```

Comment:
This avoid duplicates? I thought the last `=` wins.

Reply:
I don't know, but I imagine that without it we could end up with many duplicates of the same config, as we are using `StandaloneNodeFormula` inside dataset (re)generation code.